form/src/components/Result.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+
+const Result = ({ answer }) => {
+  const [query, setQuery] = useState("");
+
+  useEffect(() => {
```

Comment:
result shouldn't have api call logic, it'll be only display of answer(image and text)

Reply:
still missing